documentation/reference/rank-features.html

```diff
@@ -630,6 +630,29 @@ <h2 id="feature-list">Feature list</h2>
     and where highly weighted query terms has more impact, 0 otherwise
   </td></tr>
 
+<tr id="closeness(dimension,name)">
+  <td>closeness(<em>dimension</em>,<em>name</em>)</td>
+  <td>0</td>
+  <td><p>
+    Used with the NearestNeighbor query operator.
+    A number which is close to 1 when a point vector in the document is close
+    to a matching point vector in the query.
+    <ul>
+      <li> <em>dimension</em>: Specifies the dimension of <em>name</em>.
+           This must be either the string "field" or the string
+           "label".  When using "field", the name given must be a
+           field which has a multi-dimensional position attribute.
```

Comment:
I think we should mention that this must be a tensor to avoid confusion with the position type. E.g. "tensor attribute with one indexed dimension of size N, representing a point in a N-dimensional space".

Reply:
done